.github/workflows/netlify.yml

```diff
@@ -1,4 +1,4 @@
-name: Pull Request
+name: Netlify automatic reindexing with Algolia Crawler
```

Comment:
It's a bit verbose, some checks are not fitting the line, we can just put `Netlify -> Algolia Crawler` maybe?